packages/browser/src/node/commands/tab.ts

```diff
@@ -7,17 +7,16 @@ export const tab: UserEventCommand<UserEvent['tab']> = async (
   context,
   options = {},
 ) => {
-  const provider = context.provider
```

Comment:
I'd rather have it the other way. In any case, this doesn't affect the issue at all, so let's remove any refactoring here and leave only the bug fix, please

Reply:
No worries, reverted.